services/director-v2/src/simcore_service_director_v2/constants.py

```diff
@@ -21,5 +21,6 @@
 REGEX_DY_SERVICE_SIDECAR = rf"^{DYNAMIC_SIDECAR_SERVICE_PREFIX}_[a-zA-Z0-9-_]*"
 REGEX_DY_SERVICE_PROXY = rf"^{DYNAMIC_PROXY_SERVICE_PREFIX}_[a-zA-Z0-9-_]*"
 
-UNDEFINED_METADATA = "undefined-metadata"
+UNDEFINED_STR_METADATA = "undefined-metadata"
+UNDEFINED_INT_METADATA = int(1e10)
```

Comment:
Just thinking, wouldn't it be better and more in the "Principle of Least Surprise" to make rather those fields nullable? and when we really do not have value then provide NULL rather than some specific constant for different types? @GitHK @sanderegg

Reply:
@matusdrobuliak66 
 Well on the principle, I do not like the NULL value because you do not know whether it is on purpose, a bug in the code or some default value. At least with the specific value you know this was on purpose like that.

Reply:
@GitHK please send me a NULL in the wallet_id (this is how it will behave when we are for example in the osparc.io where the concept of wallets will not exist). Generally, WalletID is Positive Int (therefore even -1 is not good choice) In this use-case NULL is the right way to go. It will be stored in the database also as NULL.